activesupport/test/core_ext/time_with_zone_test.rb

```diff
@@ -482,22 +482,23 @@ def test_freeze_preloads_instance_variables
   end
 
   def test_method_missing_with_non_time_return_value
-    @twz.time.expects(:foo).returns('bar')
+    @twz.time.class.send(:define_method, :foo, -> { 'bar' })
```

Comment:
Ah, the error is probably because there's two dots. Try extracting `@twz.time` to a local `time` variable.

Sorry, I don't have a link handy. But it's the same thing as `def self.method_name`, it just allows you to define a method on an object.